helm/bundle/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -318,14 +318,15 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - directpv
+  - minio-directpv-operator-rhmp
   links:
-  - name: Directpv Operator
-    url: https://directpv-operator.domain
+  - name: Minio Directpv Operator Rhmp
+    url: https://minio-directpv-operator-rhmp.domain
   maintainers:
-  - email: cesar@minio.io
-    name: cesar
+  - email: dev@minio.io
+    name: MinIO, Inc.
   maturity: alpha
   provider:
-    name: directpv
+    name: MinIO, Inc.
```

Comment:
Do we need to have `  - name: MinIO, Inc.`?

Reply:
I am not sure, but by looking at other operator in the catalog, they do have it. Why? Is there a problem with having it?

Reply:
Oh you mean MinIO versus DirectPV, well as long as we have something that should be good enough!. What should I put?

Reply:
Ok in the meantime I will just put DirectPV and later we can modify if needed.